### PR TITLE
fixed minor error in master branch, add TODO's

### PR DESCRIPTION
**Pull request by psdh**

### CODE REVIEW

**Comment by h4ck3rk3y** [comment]
please tell more about the minor error that you fixed?

File: gowhereme/places_to_visit.py [hunk 1]
```diff
@@ -19,15 +19,12 @@
 route_rome = routerush.rome2rio
 
 
-
-
-
 def getPhoto(reference):
-    url = "https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photoreference=%s&key=%s" % (reference,
```
Comment (psdh) [inline gowhereme/places_to_visit.py]: syntax error here!

seems to have been incorrectly changed in the last commit

Comment (h4ck3rk3y) [inline gowhereme/places_to_visit.py]: probably happened while you were merging it onto master, warna production crash kar raha hota.

Comment (psdh) [inline gowhereme/places_to_visit.py]: Yes, mind merging this?

**Pull request merged**